File: src/test/java/com/github/weewar/mapviewer/service/impl/key/UnitImageKeyTest.java
package com.github.weewar.mapviewer.service.impl.key;

import com.github.weewar.mapviewer.service.impl.keys.UnitImageKey;
import nl.jqno.equalsverifier.EqualsVerifier;
import org.junit.Test;

public class UnitImageKeyTest {
    @Test
    public void verifyEqualsContract() {
        EqualsVerifier.forClass(UnitImageKey.class).verify();
    }
}

File: src/test/kotlin/com/github/weewar/mapviewer/service/impl/keys/UnitImageKeyTest.kt
package com.github.weewar.mapviewer.service.impl.keys

import nl.jqno.equalsverifier.EqualsVerifier
import org.junit.jupiter.api.Test

class UnitImageKeyTest {
    @Test
    fun `verify equals contract`() {
        EqualsVerifier.forClass(UnitImageKey::class.java).verify()
    }
}